velox/docs/develop/checklist-functions.rst

```diff
@@ -0,0 +1,75 @@
+=================================================
+Check the items when you add or modify a function
```

Comment:
Once this PR is merged, we could add this as a checklist in GitHub PR creation template, like https://github.com/prestodb/presto/blob/master/pull_request_template.md#contributor-checklist